src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25599.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if ANDROID // Facing a full exception when clicking on the already selected tab(App.Tap()) in Windows, iOS, and macOS, so excluded those platforms.
```

Comment:
Could you share the exception? Want to determinate if is related with a possible bug on Android or if we need to review something at testing stuff.

Reply:
I have revalidated the test cases once more with the latest source and I can confirm that it is now possible to click on the already selected tab using the tab title on Windows and AutomationId on iOS and Mac.
 
I have modified the test cases to run successfully in all platforms. Could you please review and let me know if you have any concerns?